packages/manager/apps/pci-ssh-keys/src/pages/ListingPage.tsx

```diff
@@ -229,7 +229,7 @@ export default function ListingPage() {
         </div>
       )}
 
-      {!isLoading && !error && (
+      {!isLoading && (!error || (error && isDiscoveryProject(project))) && (
```

Comment:
```suggestion
      {!isLoading && (!error || isDiscoveryProject(project)) && (
```

looks like it should be simplified

Reply:
I'm considering removing the error from the condition and leaving it as **{!isLoading && ()}**, but I'm not aware of all the potential errors that might occur. any suggestions are welcome :)